src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -373,7 +375,7 @@ export class ZaakCreateComponent implements OnInit, OnDestroy {
     this.toelichtingField.formControl.setValue(defaultToelichting);
     this.communicatiekanalen.subscribe((data) => {
       this.communicatiekanaalField.value(
-        data.find((c) => c.naam === ZaakCreateComponent.KANAAL_E_FORMULIER),
+        data.find((c) => c === ZaakCreateComponent.KANAAL_E_FORMULIER),
       );
     });
```

Comment:
I think you can remove this whole part if you apply the suggestion above, right?

Reply:
It seems so yes, thanks! 

In most cases the list of `communicatiekanalen` does not even include `KANAAL_E_FORMULIER`. It is only included if you create a zaak from an inbox productaanvraag. And only in that case the default selection should be `KANAAL_E_FORMULIER`. 

Does `SelectFormFieldBuilder(ZaakCreateComponent.KANAAL_E_FORMULIER)` then make sense when `communicatiekanalen` does not include `KANAAL_E_FORMULIER`? It seems to work but seems a bit weird..

Reply:
I made the suggested change in any case and it seems to work. :-)

Reply:
Ah I see what you mean. It still works in that case because the option is not in the list, causing it to show as unselected. But maybe that makes it a bit unclear that the option is not always there. I'll leave it up to you, either option is fine.

Reply:
Clear thanks! I'm staying with your suggestion and added a little code comment. Feels a little weird like this but it works so all is fine. :-)